cmd/agent/main.go

```diff
@@ -25,6 +26,7 @@ var (
 	addr            = "127.0.0.1:39000"
 	dataDir         = "./var/data"
 	logDir          = "./var/log"
+	logTrashDir     = "./var/log/trash"
```

Comment:
So you must skip trash folder specially when traversing log folder to delete logs.